lib/Controller/NavigationController.php

```diff
@@ -295,9 +291,6 @@ public function documentGet(string $id): Response {
 
 
 	/**
-	 *
-	 * // TODO: Delete the NoCSRF check
-	 *
 	 * @PublicPage
 	 * @NoCSRFRequired
```

Comment:
forgot to remove?

Reply:
No, it is required here, since this is an endpoint users can navigate to.